docs/user_guide_zh.md

```diff
@@ -285,6 +285,10 @@ Saga可通过以下任一方式进行构建：
    在 `application.yaml` 添加下面的配置项：
 
    ```yaml
+   eureka:
```

Comment:
Please add some description of this setting for the user to know how to use it.

Reply:
Added a simple description and Eureka official manual links